ctlevent.js

```diff
@@ -36,29 +42,15 @@ var CTLEvent = function(event) {
         if (xprop[i]['X-BEDEWORK-ALIAS']) {
             aliasString = xprop[i]['X-BEDEWORK-ALIAS'].values.text;
             propList = aliasString.split('/').slice(-2);
-            switch (propList[0]) {
-                case 'Category':
-                    this.category.push(propList[1]);
-                    break;
-
-                case 'Type':
-                    this.type.push(propList[1]);
-                    break;
 
-                case 'Events open to':
-                    this.eventsOpenTo.push(propList[1]);
-                    break;
-
-                case 'Group-Specific':
-                    this.groupSpecific.push(propList[1]);
-                    break;
-
-                case 'Location':
-                    this.campusLocation.push(propList[1]);
-                    break;
+            function checkPropertyName(element) {
```

Comment:
We shouldn't need to define this as a nested function - it creates unclear scoping rules, especially if it's within a `for` loop. Just define this as a static method on the class, like `CTLEvent.checkPropertyName = function(element, propItem) { ... };`.

In fact, if all it's doing is just an equality check I'm not sure we need it.